### PR TITLE
member list hover minor css update for members

### PR DESCRIPTION
**Pull request by Harshi7016**
image

I have made minor css updates

### CODE REVIEW

**Comment by Harshi7016** [comment]
I observed the last card in the page gets css issue
image

**Comment by Harshi7016** [comment]
I didn't get the time to check for this issue yet
I am working on it

File: components/member-list-item/member-list-item.module.scss [hunk 1]
```diff
@@ -33,3 +33,8 @@
     width: 28px;
   }
 }
+
+.container:hover {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  transform: translateY(-10px);
+}
```
Comment (ankushdharkar) [inline components/member-list-item/member-list-item.module.scss:40]: Hovering indicates we can do something with the card, like clicking it our picking it. Is that what we expect to allow the user?

Comment (Harshi7016) [reply components/member-list-item/member-list-item.module.scss:40]: Yeah, I mean after going through the basic structure, I observed on clicking the image we would be going to the details page.
Either that or I can just add box shadow to the card and remove the transition on hovering....
Open to suggestion

Comment (ankushdharkar) [reply components/member-list-item/member-list-item.module.scss:40]: So, we are doing something on clicking? Got it. Thanks for explaining @Harshi7016 :)

**Pull request merged**